docs/usage/faq_and_troubleshooting.md

```diff
@@ -0,0 +1,354 @@
+│       ├── .command.err    # stderr log messages
+│       ├── .command.out    # stdout log messages
+│       ├── .command.trace  # Compute resource usage stats
+│       ├── .exitcode       # Exit code
+│       ├── <...>           # Input/output files or directories
+│       └── versions.yml    # WiGiTS tool version
+<...>
+```
+
+The `work/` directory can be hard to navigate due to the `<short_hash>/<long_hash>` structure. These hashes are shown
+(truncated) in the console while running `oncoanalyser` (but can also be found in the `.nextflow.log` files):
+
+```shell
+[e5/f6e2e8] process > NFCORE_ONCOANALYSER:WGTS:REDUX_PROCESSING:REDUX (<group_id>_<sample_id>)     [100%] 2 of 2 ✔
+```
+
+Otherwise, you can use a utility like [tree](<https://en.wikipedia.org/wiki/Tree_(command)>) to show the directory
+structure, which allows you to manually find the target process directory.
+
+## Saving logs from the `work/` directory
+
+To save logs to the final output directory (i.e. path provided to `--outdir`), we can provide the below
+[afterScript](https://www.nextflow.io/docs/latest/reference/process.html#afterscript) directive in a config file:
+
+```groovy
+// Adapted from this GitHub issue: https://github.com/nextflow-io/nextflow/issues/1166
+process.afterScript = {
+    // params.outdir: --outdir arg
+    // meta.key: sample_id from the sample sheet
+    log_dir = "${params.outdir}/${meta.key}/logs"
+
+    // task.process: name of the process
+    // meta.id: concatenation of the group_id and sample_id from the sample sheet
+    dest_file_prefix = "${log_dir}/${task.process}.${meta.id}"
+
+    // The value of afterScript is simply a bash command as a string
+    cmd =  "mkdir -p ${log_dir}; "
+    cmd += "for file in .command.{sh,log}; do cp \$file ${dest_file_prefix}\${file}; done"
+    cmd
+}
+```
+
+The above afterScript directive will copy `.sh` and `.log` files from the `work/` directory for every process. Each
+destination file will have the below example path:
+
+```shell
+outdir/coloMini/logs/NFCORE_ONCOANALYSER:WGTS:REDUX_PROCESSING:REDUX.coloMini_coloMiniT.command.log
+```
+
+## Resuming runs in Google Batch?
+
+When resuming with runs in Google Batch (using `-resume`), you will need to enable overwriting of the `pipeline_info`
+files (performance and run time stats) as shown below. By default, these files are not overwritten thus preventing
+`oncoanalyser` from starting.
+
+```groovy
+timeline {
+    enabled   = true
+    overwrite = true
+    file      = "${params.outdir}/pipeline_info/execution_timeline.html"
+}
+
+report {
+    enabled   = true
+    overwrite = true
+    file      = "${params.outdir}/pipeline_info/execution_report.html"
+}
+
+trace {
+    enabled   = true
+    overwrite = true
+    file      = "${params.outdir}/pipeline_info/execution_trace.txt"
+}
+
+dag {
+    enabled   = true
+    overwrite = true
+    file      = "${params.outdir}/pipeline_info/pipeline_dag.svg"
+}
+```
```

Comment:
All of this documentation is super nice and quite clear, I like it, but I feel that this is not very pipeline specific and would benefit the whole community if it could make it on the website

Reply:
Agreed, let's see how we can migrate some of this documentation into a common area for the community in the next release! I've created an issue to track:

- https://github.com/nf-core/oncoanalyser/issues/186

Reply:
/cc @luan-n-nguyen @rayanhassaine

Reply:
Hi @maxulysse, I'd like to open a PR to integrate relevant parts of the documentation into a common location within nf-core - do you have any suggestions for where this would best fit? First thought would be under the [troubleshooting section](https://nf-co.re/docs/usage/troubleshooting)